src/System.Memory/ref/System.Memory.cs

```diff
@@ -184,51 +180,50 @@ public partial struct Enumerator
             public bool MoveNext() { throw null; }
         }
     }
-    public ref partial struct SequenceReader<T> where T : unmanaged, System.IEquatable<T>
+    public static partial class SequenceReaderExtensions
     {
-        public SequenceReader(System.Buffers.ReadOnlySequence<T> buffer) { }
-        public bool IsLastSegment { get { throw null; } }
-        public bool End { get { throw null; } }
-        public ReadOnlySequence<T> Sequence { get { throw null; } }
-        public SequencePosition Position { get { throw null; } }
-        public ReadOnlySpan<T> CurrentSpan { get { throw null; } }
-        public int CurrentSpanIndex { get { throw null; } }
-        public ReadOnlySpan<T> UnreadSpan { get { throw null; } }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out short value) { throw null; }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out int value) { throw null; }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out long value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out short value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out int value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out long value) { throw null; }
+        public static bool TryRead<T>(this ref System.Buffers.SequenceReader<byte> reader, out T value) where T : struct { throw null; }
```

Comment:
`T : struct` instead of `unmanaged`?

cc @weshaggard

Reply:
Apparently a bug in the generation code. I'll fix it.

Reply:
Manually changed for now.

Reply:
https://github.com/dotnet/arcade/issues/1299